doc/classes/ShortCut.xml

```diff
@@ -21,17 +25,21 @@
 			<argument index="0" name="event" type="InputEvent">
 			</argument>
 			<description>
+				Returns true if Shortcut's [InputEvent] equals [code]event[/code].
 			</description>
 		</method>
 		<method name="is_valid" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether or not this Shortcut is valid.
```

Comment:
Per doc guidelines, we're trying to avoid "whether or not" construction. Use "If [code]true[/code] this Shortcut is valid."